python/GafferSceneUI/MotionPathUI.py

```diff
@@ -0,0 +1,191 @@
+##########################################################################
+#
+#  Copyright (c) 2021, Image Engine Design Inc. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of Image Engine Design Inc nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import Gaffer
+import GafferScene
+
+import GafferUI
+
+Gaffer.Metadata.registerNode(
+
+	GafferScene.MotionPath,
+
+	"description",
+	"""
+	Creates a motion path curve over the specified frame range for each filtered location.
+	Note the output scene will be isolated to the matching locations only.
+	""",
+
+	plugs={
+
+		"start" : [
+
+			"description",
+			"""
+			The first frame of motion tracking can be specified relative to the current frame or as an absolute value
+			""",
+
+			"plugValueWidget:type", "GafferSceneUI.MotionPathUI._FramePlugValueWidget",
+
+		],
+
+		"start.mode" : [
+
+			"description",
+			"""
+			Controls whether the "start" frame is relative to the current frame or an absolute value 
```

Comment:
Let's put periods at the end of each of these sentences to match the prevailing doc style, and use backticks for plug names to improve the formatting e.g. `start.frame`.

Reply:
Addressed in 918780477cf7dcc9fd7c86be7a9fcb8dc9fd9c54